juriscraper/oral_args/united_states/federal_appellate/ca11.py

```diff
@@ -6,47 +6,28 @@
 Date created: 28 Aug 2018
 """
 
-from datetime import datetime
+from juriscraper.OralArgumentSiteLinear import OralArgumentSiteLinear
 
-from juriscraper.OralArgumentSite import OralArgumentSite
 
-
-class Site(OralArgumentSite):
+class Site(OralArgumentSiteLinear):
     base_url = "https://www.ca11.uscourts.gov/oral-argument-recordings"
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
         self.url = self.base_url
         self.back_scrape_iterable = [i for i in range(0, 52)]
-        self.base_path = "//tr[not(th)]"
-
-    def _get_download_urls(self):
-        path = f"{self.base_path}//td[5]//@href"
-        return list(self.html.xpath(path))
-
-    def _get_case_names(self):
-        path = f"{self.base_path}//td[2]/text()"
-        return list(self.html.xpath(path))
 
-    def _get_case_dates(self):
-        path = f"{self.base_path}//td[3]/text()"
-        return list(map(self._return_case_date, self.html.xpath(path)))
-
-    @staticmethod
-    def _return_case_date(s):
-        return datetime.strptime(s.strip(), "%Y-%m-%d").date()
-
-    def _get_docket_numbers(self):
```

Comment:
what do these fixes look like?

Reply:
The scraper produces the same data as before, that why I didn't have to alter the example files